From 2621c147fb65a4034fd186b07792634d26488b5f Mon Sep 17 00:00:00 2001
From: "Brian C. Lane" <bcl@redhat.com>
Date: Wed, 16 Apr 2014 17:58:29 -0700
Subject: [PATCH 96/99] tests: Make t4100 xfs filesystem larger and sparse

Newer versions of mkfs.xfs appear to be using more blocks for the
journal, causing t4100-* tests to fail. Make the image larger.

* tests/t4100-dvh-partition-limits.sh: Make the xfs image larger
* tests/t4100-msdos-partition-limits.sh: Make the xfs image larger
---
 tests/t4100-dvh-partition-limits.sh   | 2 +-
 tests/t4100-msdos-partition-limits.sh | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/t4100-dvh-partition-limits.sh b/tests/t4100-dvh-partition-limits.sh
index a1bc69a..fb10025 100755
--- a/tests/t4100-dvh-partition-limits.sh
+++ b/tests/t4100-dvh-partition-limits.sh
@@ -37,7 +37,7 @@ mp=`pwd`/mount-point
 n=4096
 
 # create an XFS file system
-dd if=/dev/zero of=$fs bs=1MB count=2 seek=20 || fail=1
+dd if=/dev/null of=$fs bs=1MB seek=50 || fail=1
 mkfs.xfs -f -q $fs || fail=1
 mkdir "$mp" || fail=1
 
diff --git a/tests/t4100-msdos-partition-limits.sh b/tests/t4100-msdos-partition-limits.sh
index 65bda25..af7907f 100755
--- a/tests/t4100-msdos-partition-limits.sh
+++ b/tests/t4100-msdos-partition-limits.sh
@@ -37,7 +37,7 @@ mp=`pwd`/mount-point
 n=4096
 
 # create an XFS file system
-dd if=/dev/zero of=$fs bs=1MB count=2 seek=20 || fail=1
+dd if=/dev/null of=$fs bs=1MB seek=50 || fail=1
 mkfs.xfs -f -q $fs || fail=1
 mkdir "$mp" || fail=1
 
-- 
1.9.0

